.gitignore

```diff
@@ -1,4 +1,2 @@
 /.idea/
-/gradle/
```

Comment:
I'd rather not include the jar here but only the properties file - any update of Gradle now and it will be another binary blob diff.